src/runtime_src/core/common/api/xrt_module.cpp

```diff
@@ -346,7 +348,7 @@ class module_impl
   // @param patch - patch value
   // @Return true if symbol was patched, false otherwise  //
   virtual bool
-  patch(uint8_t*, const std::string&, uint64_t)
+  patch(uint8_t*, const std::string&, uint64_t, patcher::buf_type)
```

Comment:
document above not updated, same goes multiple places

Reply:
Thanks for catch that. Fixed.